clang/lib/DPCT/ASTTraversal.cpp

```diff
@@ -12329,6 +12333,18 @@ void KernelFunctionInfoRule::runRule(const MatchFinder::MatchResult &Result) {
     if (NameMap != AttributesNamesMap.end())
       emplaceTransformation(new ReplaceToken(MemberName.getBeginLoc(),
                                              std::string(NameMap->second)));
+
+  } else if (auto *CallNode =
+                 getNodeAsType<CallExpr>(Result, "cuFuncSetAttribute")) {
+    std::string FuncName =
+        CallNode->getDirectCallee()->getNameInfo().getName().getAsString();
+    auto Msg = MapNames::RemovedAPIWarningMessage.find(FuncName);
+    // TODO: Handle assignments
```

Comment:
it is similar  to line 12312, can add a `if (isAssigned(C)) {`.

Reply:
I think so, cuFuncSetAttribute has a return value, it may be checked in the user's code.